src/shared/components/structureViewer/StructureViewerPanel.tsx

```diff
@@ -508,6 +530,24 @@ export default class StructureViewerPanel extends React.Component<IStructureView
         }
     }
 
+    @computed get structureViewerBounds(): {width: number|string, height: number|string} {
+        let width: number|string;
+        let height: number|string;
+
+        // if 3Dmol container div is not initialized yet, just set to a default value: width=auto; height=350
+        // otherwise toggle the size
+        if (this.isIncreasedSize) {
```

Comment:
mobx registers observables per-execution. So whatever it sees on one execution, is listening for and then when it executes again, it forgets what happened last time and just remembers what its encountered this time.

So the first time the function executes, its before the 3dMolDiv ref is called, so it doesnt see the branch that has `isIncreasedSize` and doesnt register it. In the way you've written it in the PR, it always sees `isIncreasedSize`